### PR TITLE
Added POM modules

### PR DESCRIPTION
**Pull request by jeger**
Modules added:
-client
-server
-shared
No changes in functionality.

### CODE REVIEW

File: client/dependency-reduced-pom.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tomeks86) [inline client/dependency-reduced-pom.xml]: What is this reduced-pom?

File: client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>client</artifactId>
+    <version>0.2.0.1</version>
+
+
+    <parent>
+        <artifactId>battleships</artifactId>
+        <groupId>com.ostatki</groupId>
+        <version>0.2.0.1</version>
+        <relativePath>../pom.xml</relativePath>
```
Comment (tomeks86) [inline client/pom.xml]: check if this is necessary

File: server/pom.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>server</artifactId>
+    <version>0.2.0.1</version>
+
+    <parent>
+        <artifactId>battleships</artifactId>
+        <groupId>com.ostatki</groupId>
+        <version>0.2.0.1</version>
+        <relativePath>../pom.xml</relativePath>
```
Comment (tomeks86) [inline server/pom.xml]: check if this is necessary

File: shared/pom.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>shared</artifactId>
+    <version>0.2.0.1</version>
+
+
+    <parent>
+        <artifactId>battleships</artifactId>
+        <groupId>com.ostatki</groupId>
+        <version>0.2.0.1</version>
```
Comment (tomeks86) [inline shared/pom.xml:14]: hint to update version number - use:
mvn versions:set -DnewVersion=0.2.5

**Review by tomeks86 — DISMISSED** [review]
check what is the client-reduced-pom.xml

File: .gitignore [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /.idea
-/target
+/**/target
 *.class
 *.iml
 *.swp
```
Comment (makoJohn) [inline .gitignore:5]: new line

**Review by makoJohn — APPROVED** [review]
OK

File: shared/src/main/java/placement/model/ship/PlacedShip.java [hunk 6]
```diff
@@ -33,5 +33,3 @@ public DirectedShip direct(Direction direction) {
     }
 
 }
-
```
Comment (JaroslawZaczyk) [inline shared/src/main/java/placement/model/ship/PlacedShip.java:36]: good change! congratulations :)

**Review by JaroslawZaczyk — APPROVED** [review]
Looks good, but in the case of problems, please go to the author, because my knowledge about this code is not very well, so I can't decide about anything.

**Pull request merged**